src/axes/categoryAxis.ts

```diff
@@ -74,6 +76,81 @@ module Plottable.Axes {
     }
 
     /**
+     * Gets the current alignment. If not set, null will be returned.
+     */
+    public tickTextAlignment(): string;
+    /**
+     * Set alignment of tick labels. Must be one of "left", "right",
+     * or "center", or a falsy value. Setting to a falsy value will
+     * reset tick label alignment to the default alignment.
+     *
+     * Note that alignment is applied before rotation (see "{@link
+     * tickLabelAngle}").
+     * 
+     * @param {String} tickTextAlignment One of "left", "right", "center",
+     * or a falsy value.
+     *
+     * @returns {Category} The calling Category.
+     */
+    public tickTextAlignment(tickTextAlignment: string): this;
+    public tickTextAlignment(tickTextAlignment?: string): any {
+      if (arguments.length === 0) {
```

Comment:
Instead of checking `arguments.length`, let's use the idiom used elsewhere in the codebase and check if the `tickTextAlignment` param is specified or not. ([Similar to here](https://github.com/palantir/plottable/blob/8e736c4367f9e2ca56b6edb15731a16efe014dc8/src/plots/stackedAreaPlot.ts#L52) for example.)